src/renderer/coremods/badges/badge.tsx

```diff
@@ -104,7 +105,7 @@ const openExternal = (url: string): Promise<void> =>
 
 // todo: make global (configurable?) variables for these
 const openContributorsPage = (): Promise<void> =>
-  openExternal("https://replugged.dev/contributors");
+  openExternal(`https://${generalSettings.get("apiUrl", "https://replugged.dev")}/contributors`);
```

Comment:
Fallback is not needed since you hve the default settings.
```suggestion
  openExternal(`https://${generalSettings.get("apiUrl")}/contributors`);
```